libraries/src/Installer/InstallerHelper.php

```diff
@@ -395,7 +395,7 @@ public static function isChecksumValid($packagefile, $updateObject)
 				$hashRemote  = $updateObject->$hash->_data;
 				$hashOnFile  = true;
 
-				if ($hashPackage !== $hashRemote)
+				if (strtolower($hashPackage) !== strtolower($hashRemote))	
```

Comment:
The hash_file function returns only lowercase characters so we need to convert them.
```suggestion
				if ($hashPackage !== strtolower($hashRemote))	
```